app/validators/application_fulfilment_validator.rb

```diff
@@ -30,4 +30,8 @@ def perform_validations # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
   def ioj_present?
     record.ioj.present? && record.ioj.types.any?
   end
+
+  def means_record_present?
+    record.income.present? && record.income.employment_status.include?('not_working')
```

Comment:
Does `income.employment_status` need a `nil` check as well? The schema at the moment doesn't enforce its presence but it is validated for correctness in `employment_status_form.rb`

Reply:
I can do to be double safe. But it is the first step in income, so if record.income.present? is true, the user has landed on the employment_status page and it's an empty array. I get your point though - I'll put it in in case the first page changes/the user can somehow skip p1 of income